scipy/optimize/_hessian_update_strategy.py

```diff
@@ -0,0 +1,338 @@
+"""quasi-Newton update strategies."""
+
+from __future__ import division, print_function, absolute_import
+import numpy as np
+from numpy.linalg import norm
+from scipy.linalg import get_blas_funcs
+
+class HessianUpdateStrategy(object):
+    """Virtual interface for implementing Hessian update strategies.
+
+    Should implement four methods: ``initialize``, ``update``,
```

Comment:
Again minor, but an interface doesn't implement methods, but rather derived classes implement them. Something like that "Four methods should be implemented in derived classes: ...."

Reply:
Ok, I agree with you there.